en/docs/try-out/payment-initiation-service-flow.md

```diff
@@ -0,0 +1,601 @@
+This page provides instructions to use the NextGenPsd2XS2AFramework API to provide the Payment Initiation Service (PIS).
```

Comment:
capitalize PSD. check other places as well

Reply:
Fixed in 7bfa39b2fba54a9ecb39b651ec24854dea85354f